lib/components/SInputSelect.vue

```diff
@@ -80,10 +69,11 @@ function emitChange(e: any): void {
   <SInputBase
     class="SInputSelect"
     :class="classes"
-    :label="label"
-    :note="note"
-    :info="info"
-    :help="help"
+    :size
+    :label
+    :note
+    :info
+    :help
     :check-icon="checkIcon"
     :check-text="checkText"
     :check-color="checkColor"
```

Comment:
not sure, but most likely these can be changed to shorthand too 😅

Reply:
Ah, good point. Gonna do it!

Reply:
Done!